clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -526,6 +526,11 @@ llvm::Type *CodeGenTypes::ConvertType(QualType T) {
       }
     case BuiltinType::SveCount:
       return llvm::TargetExtType::get(getLLVMContext(), "aarch64.svcount");
+    case BuiltinType::ArmMFloat8:
+      ResultType =
+          llvm::VectorType::get(llvm::Type::getInt8Ty(getLLVMContext()),
+                                llvm::ElementCount::getFixed(1));
+      break;
```

Comment:
Not sure how good an idea this is but what about updating `getBuiltinVectorTypeInfo` so the code above can be used? I mean, with the new definition mfloat8_t is effectively a vector type of sorts.